acme/tests/messages_test.py

```diff
@@ -467,5 +468,18 @@ def test_to_partial_json(self):
         })
 
 
+class JWSPayloadRFC8555Compliant(unittest.TestCase):
+    """Test for RFC8555 compliance of JWS generated from resources/challenges"""
+    def test_message_payload(self):
+        from acme.messages import NewAuthorization
+
+        new_order = NewAuthorization()
+        new_order.le_acme_version = 2
+
+        jobj = new_order.json_dumps(indent=2).encode()
+        # RFC8555 states that challenge requests must have an empty payload.
```

Comment:
I think you accidentally switched the comments in the two tests you added as they both seem more appropriate in the other location.

If so, I'd change this comment to say "...challenge responses..." when it's moved to the other file.

Reply:
Fixed.